modules/nf-core/picard/markduplicates/main.nf

```diff
@@ -24,23 +25,27 @@ process PICARD_MARKDUPLICATES {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+    def suffix = task.ext.suffix    ?: "${reads.getExtension()}"
+    def reference = fasta ? "--REFERENCE_SEQUENCE ${fasta}" : ""
+    def create_index = ( suffix == "bam" )? "--CREATE_INDEX" : ""
```

Comment:
`--create_index` shouldn't be added in the module, but added by the user through `ext.args`

Reply:
removed it